plugin/githubinator.vim

```diff
@@ -64,23 +75,23 @@ function! GithubOpenURL() range
     endif
 endfunction
 
-function! GithubCopyURL() range
+function! s:github_copy_url()
     let l:final_url = s:generate_url()
 
     if executable('pbcopy')
-        call system("echo " . l:final_url . " | pbcopy")
+        call system('echo ' . l:final_url . ' | pbcopy')
     elseif executable('xsel')
-        call system("echo " . l:final_url . " | xsel -b")
+        call system('echo ' . l:final_url . ' | xsel -b')
     else
-        echoerr "githubinator: no `copy-to-clipboard` command found."
+        echoerr 'githubinator: no `copy-to-clipboard` command found.'
         return
     endif
 
-    echom "Githubinator: URL copied to clipboard."
+    echom 'Githubinator: URL copied to clipboard.'
 endfunction
 
-vnoremap <silent> <Plug>(githubinator-open) :<C-U>call GithubOpenURL()<CR>
-vnoremap <silent> <Plug>(githubinator-copy) :<C-U>call GithubCopyURL()<CR>
+vnoremap <silent> <Plug>(githubinator-open) :<C-U>call <SID>github_open_url()<CR>
+vnoremap <silent> <Plug>(githubinator-copy) :<C-U>call <SID>github_copy_url()<CR>
```

Comment:
@mopp while you're at it, can you also make a quick little change to incorporate #11 ?

Reply:
ok, I can.
This PR contains much changes.
But, I don't take care.